src/Features/Core/Portable/Completion/Providers/AbstractKeywordCompletionProvider.cs

```diff
@@ -86,6 +86,11 @@ protected virtual CompletionItem CreateItem(RecommendedKeyword keyword, TextSpan
             int position,
             CancellationToken cancellationToken)
         {
+            if (!document.SupportsSyntaxTree)
```

Comment:
This type should only be derived by C# and VB implementations.  They would always have syntax trees.